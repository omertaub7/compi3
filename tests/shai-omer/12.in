// simple while-else
void main () {
	int x = 3;
	while (x > 0) {
		printi(x);
		x = x - 1;
	}
	else {
		print("GOOD");
	}
}