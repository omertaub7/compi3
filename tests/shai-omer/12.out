3
2
1
GOOD
